src/components/Tutorial/TutorialStepAutoExpand.tsx

```diff
@@ -86,9 +86,9 @@ const TutorialStepAutoExpand = () => {
               <>
                 {' '}
                 to hide
-                {(isCursorLeaf ? cursorValue : cursorChildValue).length === 0 && ' the empty '} subthought
+                {(isCursorLeaf ? cursorValue : cursorChildValue)?.length === 0 && ' the empty '} subthought
                 {ellipsize(
-                  isCursorLeaf && cursorValue.length > 0
+                  isCursorLeaf && cursorValue && cursorValue.length > 0
```

Comment:
I'm not sure about places like this
previously we assumed `string` and it was wise to be like `length > 0`
but now it's `string | undefined` and `isCursorLeaf && cursorValue`  could be viable, though implicit. Maybe sounds hacky, but `cursorValue && cursorValue.length > 0` is so ugly and meaningless

Reply:
This approach looks fine, just make sure to check for undefined explicitly.